src/main/java/lotto/adapter/inbound/cli/CliInputAdapter.java

```diff
@@ -0,0 +1,93 @@
+package lotto.adapter.inbound.cli;
+
+import lotto.application.dto.request.ManageLottoRequest;
+import lotto.application.dto.response.ManageLottoResponse;
+import lotto.application.port.inbound.CreateLottoUseCase;
+import lotto.application.port.inbound.InputAdapter;
+import lotto.application.port.inbound.InputPort;
+import lotto.application.port.inbound.ManageLottoUseCase;
+import lotto.application.port.outbound.OutputPort;
+import lotto.application.validator.LottoInputValidator;
+import lotto.domain.lotto.Lotto;
+import lotto.domain.lotto.WinningLotto;
+import lotto.domain.cost.Cost;
+import lotto.domain.rank.RankResult;
+import lotto.domain.rank.vo.RevenueRate;
+import lotto.infrastructure.constant.PromptMessage;
+
+import java.util.List;
+
+public class CliInputAdapter implements InputAdapter {
+    private final ManageLottoUseCase manageLottoUseCase;
+    private final CreateLottoUseCase createLottoUseCase;
+    private final OutputPort outputPort;
+    private final InputPort inputPort;
+    private final LottoInputValidator lottoInputValidator;
+
+    public CliInputAdapter(ManageLottoUseCase manageLottoUseCase,
+                           CreateLottoUseCase createLottoUseCase,
+                           OutputPort outputPort, InputPort inputPort,
+                           LottoInputValidator lottoInputValidator) {
+        this.manageLottoUseCase = manageLottoUseCase;
```

Comment:
```suggestion
    public CliInputAdapter(
         ManageLottoUseCase manageLottoUseCase,
         CreateLottoUseCase createLottoUseCase,
         OutputPort outputPort, InputPort inputPort,
         LottoInputValidator lottoInputValidator
    ) {
        this.manageLottoUseCase = manageLottoUseCase;
```

취향 차이라고 생각하지만 혹시 이런 스타일은 어떤가요? 개인적으로 가독성이 더 좋다고 생각해서요!

Reply:
제안해주신 방법이 훨씬 깔끔한 것 같아요~! 이번에 반영해보겠습니다!